content/influxdb/cloud/reference/internals/ttbr.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Time To Become Readable
+description: >
+  **Time To Become Readable (TTBR)** is the delay between when you write data to
+  InfluxDB Cloud and and when that data becomes queryable.
+  TTBR is variable and is affected by many factors.
+menu:
+  influxdb_cloud_ref:
+    name:  Time To Become Readable
+    parent: InfluxDB Cloud internals
+weight: 102
+influxdb/cloud/tags: [write, query]
+related:
+  - /influxdb/cloud/write-data/
+  - /influxdb/cloud/query-data/
+---
+
+**Time To Become Readable (TTBR)** is the delay between when you write data to
+InfluxDB Cloud and and when that data becomes queryable.
+TTBR is variable and is affected by many factors.
+
+- [How write requests work in the InfluxDB Cloud API](#how-write-requests-work-in-the-influxdb-cloud-api)
+- [Flux vs InfluxQL](#flux-vs-influxql)
+- [InfluxDB Cloud TTBRs](#influxdb-cloud-ttbrs)
+
+## How write requests work in the InfluxDB Cloud API
+
+Whenever you send a write request to the `/api/v2/write` endpoint, the following occurs:
+
+- Validates the request and queues the write.
+- If the write is queued, responds with an HTTP 204 status code.
+- Handles the write asynchronously and reaches eventual consistency.
+
+_For more information, see [`/api/v2/write` documentation](/influxdb/cloud/api/#operation/PostWrite)._
+
+{{% note %}}
+The returned 204 status code code does not mean that the point is queryable;
```

Comment:
This is a good thing to have in there, might even be worth cross-linking to https://docs.influxdata.com/influxdb/cloud/write-data/troubleshoot/#handle-write-and-delete-responses? That page gives quite a solid overview of the write process and why a write might fail even though it was a 204.